### PR TITLE
chore(docs): update tabs docs examples for additional ARIA markup

### PR DESCRIPTION
**Pull request by tmorehouse**
### Describe the PR

Add aria roles to contentless tabs examples

### PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Enhancement
- [ ] ARIA accessibility
- [x] Documentation update
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** (check one)

- [x] No
- [ ] Yes (please describe)

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (i.e. `[...] (fixes #xxx[,#xxx])`, where "xxx" is the issue number)
- [x] It should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] The title should follow the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. `fix(alert): not alerting during SSR render`, `docs(badge): update pill examples`, `chore(docs): fix typo in README`, etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [x] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] Includes any needed TypeScript declaration file updates
- [ ] New/updated tests are included and passing (if required)
- [x] Existing test suites are passing
- [x] CodeCov for patch has met target
- [x] The changes have not impacted the functionality of other components or directives
- [x] ARIA Accessibility has been taken into consideration (Does it affect screen reader users or keyboard only users? Clickable items should be in the tab index, etc.)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: src/components/tabs/README.md [hunk 1]
```diff
@@ -57,17 +57,17 @@ When `<b-tabs>` is in `card` mode, each `<b-tab>` sub-component will automatical
   <b-card no-body>
     <b-tabs card>
       <b-tab no-body title="Picture 1">
-        <b-card-img bottom src="https://picsum.photos/600/200/?image=21"></b-card-img>
+        <b-card-img bottom src="https://picsum.photos/600/200/?image=21" alt="img"></b-card-img>
```
Comment (jackmu95) [inline src/components/tabs/README.md:60]: Maybe "Image 21"?

Comment (tmorehouse) [reply src/components/tabs/README.md:60]: Maybe... just as long as there is an `alt` attribute

Comment (tmorehouse) [reply src/components/tabs/README.md]: Should probably make sure all the card images in the card docs also have `alt` attributes

Comment (tmorehouse) [reply src/components/tabs/README.md]: Card docs updated

**Pull request merged**